app/navbar.py

```diff
@@ -58,6 +58,26 @@ def Navbar(dropdown=False):
         className="mx-auto order-0",
         style={'color': 'white', 'align': 'center', 'justify': 'center'})
 
+    # Alert monitoring screen, to be displayed in CODIS
+    alert_screen = dbc.NavLink(
```

Comment:
Last low value-added comment I promise: what do you think of renaming this object "alert_screen_button" or something equivalent so that we don't get confused with the functions and component in the `alert_screen.py` file?

Reply:
done !